# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: docshell/test/browser/browser.ini
# Commit: 4b76024db21c
# Full Hash: 4b76024db21cff006a7424a35c412c6e0a153eb0
# Author: Kris Maglione <maglione.k@gmail.com>
# Date: 2020-09-22 03:47:42
# Regressor Bug: 1655866
# File Overlap Count: 1
# Description:
#   Bug 1655866: Part 5 - Use native PermitUnload implementation from front-end code. r=nika,mconley
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D88318
# ==============================================================================

diff -r 55fb069319f1 -r 4b76024db21c docshell/test/browser/browser.ini
--- a/docshell/test/browser/browser.ini	Mon Sep 21 22:41:01 2020 +0000
+++ b/docshell/test/browser/browser.ini	Mon Sep 21 22:41:04 2020 +0000
@@ -39,11 +39,6 @@
   file_bug1328501.html
   file_bug1328501_frame.html
   file_bug1328501_framescript.js
-  file_bug1415918_beforeunload_2.html
-  file_bug1415918_beforeunload_3.html
-  file_bug1415918_beforeunload_iframe_2.html
-  file_bug1415918_beforeunload_iframe.html
-  file_bug1415918_beforeunload.html
   file_bug1543077-1-child.html
   file_bug1543077-1.html
   file_bug1543077-2-child.html